src/main/java/moze_intel/projecte/utils/WorldHelper.java

```diff
@@ -510,7 +510,7 @@ public static void igniteNearby(World world, EntityPlayer player)
 	{
 		for (BlockPos pos : BlockPos.getAllInBoxMutable(new BlockPos(player).add(-8, -5, -8), new BlockPos(player).add(8, 5, 8)))
 		{
-			if (world.rand.nextInt(128) == 0 && world.isAirBlock(pos))
+			if (world.rand.nextInt(32) == 0 && world.isAirBlock(pos))
```

Comment:
This is a balance change?

Reply:
It was only to compensate for the less ticks it'll be doing, if we find a better solution this could be reverted